### PR TITLE
Move to agilecontent.com

### PR DESCRIPTION
**Pull request by TobbeEdgeware**
Preparation to move this repo to https://github.com/agilecontent/efp since unit-x, the research part of Edgeware, and the whole Edgeware is now a part of "Agile Content".

### CODE REVIEW

File: ElasticFrameProtocol.h [hunk 1]
```diff
@@ -7,10 +7,10 @@
 //  | |____ | || (_| |\__ \| |_ | || (__ | |   | |  | (_| || | | | | ||  __/
 //  |______||_| \__,_||___/ \__||_| \___||_|   |_|   \__,_||_| |_| |_| \___|
 //                                                                  Protocol
-// UnitX @ Edgeware AB 2021
+// UnitX @ Edgeware AB 2021 => agilecontent 2022
```
Comment (magnusedgeware) [inline ElasticFrameProtocol.h:10]:
```suggestion
// UnitX @ Edgeware AB 2021 => AgileContent 2022
```

File: ElasticFrameProtocol.h [hunk 2]
```diff
@@ -203,7 +203,7 @@ class ElasticFrameProtocolContext {
  *
  * \author UnitX
  *
- * Contact: https://github.com/andersc or https://github.com/Unit-X
+ * Contact: https://github.com/andersc or https://github.com/agilecontent
```
Comment (magnusedgeware) [inline ElasticFrameProtocol.h:206]: Is it time to let Anders C go now?

**Review by magnusedgeware — APPROVED** [review]
LGTM

**Review by tomasoh-edgeware — APPROVED** [review]

File: ElasticFrameProtocol.h (hunk 1 above)
Comment (TobbeEdgeware) [reply ElasticFrameProtocol.h:10]: It should be Agile Content with a space. I'll change.

**Pull request merged**